app.js

```diff
@@ -54,16 +56,19 @@ app.use((req, res, next) => {
   next(err);
 });
 
-// development error handler
-// will print stacktrace
+// error handler
+// will print stacktrace if not production
 app.use((err, req, res, next) => {
   res.status(err.status || 500);
   res.json({
+    status: 'failure',
```

Comment:
Why are you adding the `status` attribute to the JSON error response object?
This is not in sync with the team's convention of displaying error messages.